website/docs/docs/collaborate/git/version-control-basics.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Version control basics"
+id: version-control-basics
+description: "Leverage git to version control your code."
+sidebar_label: "Version control basics"
+---
+
+# Version control basics
+
+:::info 📌 
+
+To use version control, make sure you are connected to a Git repository in the Cloud IDE. To use version control, make sure you are connected to a Git repository in the IDE.
+
+:::
+
+A [version control](https://en.wikipedia.org/wiki/Version_control) system allows you and your teammates to work collaboratively, safely, and simultaneously on a single project. Version control helps you track all the code changes made in the [Cloud IDE](/docs/develop/develop-in-the-cloud).
+
+When you develop in the Cloud IDE, you can leverage Git directly to version control your code from your browser. This means you can branch, commit, push, and pull code with a couple of clicks - no command line required!
+
+You can create a separate branch to develop and make changes. The changes you make aren’t merged into the main branch unless it successfully passes tests. This helps keep the code organized and improves productivity by making the development process smooth.
+
+![Git overview](https://docs.getdbt.com/img/docs/dbt-cloud/cloud-ide/git-overview.png)
+
+## Git overview
+
+Check out some common git terms below that you may encounter when developing:
+
+| Name | Definition |
+| --- | --- |
+| Repository or repo | A repository is a directory that stores all the files, folders, and content needed for your project. You can think of this as an object database of the project, storing everything from the files themselves to the versions of those files, commits, and deletions. Repositories are not limited by user and can be shared and copied. |
+| Branch | A branch is a parallel version of a repository. It is contained within the repository but does not affect the primary or main branch allowing you to work freely without disrupting the live version. When you've made the changes you want to make, you can merge your branch back into the main branch to publish your changes |
+| Checkout | The checkout command is used to create a new branch, change your current working branch to a different branch, or switch to a different version of a file from a different branch. |
+| Commit | A commit is a user’s change to a file (or set of files). When you make a commit to save your work, Git creates a unique ID that allows you to keep a record of the specific changes committed along with who made them and when. Commits usually contain a commit message which is a brief description of what changes were made. |
+| Main | The primary, base branch of all repositories. All committed and accepted changes should be on the Main (or master) branch. In the Cloud IDE, the “Main” branch will be read-only. This is because any changes/edits to code cannot and should not be made directly in the base branch. A new branch should be created in order to make any changes to your project |
```

Comment:
```suggestion
| Main | The primary, base branch of all repositories. All committed and accepted changes should be on the `Main` (or `master`) branch. In the Cloud IDE, the `Main` branch is read-only. This is because any changes/edits to code cannot and should not be made directly in the base branch. A new branch should be created in order to make any changes to your project |
```

Reply:
https://www.theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main 

2 suggs: 

- sugg match casing for "main"  
- suggested removing "master" bc wanted to lean toward inclusive language 🤷‍♀️